src/openrct2/ride/Track.cpp

```diff
@@ -734,27 +761,33 @@ static bool track_add_station_element(sint32 x, sint32 y, sint32 z, sint32 direc
                     ride->station_starts[stationIndex].x = (x >> 5);
                     ride->station_starts[stationIndex].y = (y >> 5);
                     ride->station_heights[stationIndex] = z;
```

Comment:
misaligned with assignment above

Reply:
Not sure why this pops up. I have fixed this and it appears aligned in the IDE.